src/Oliver.Client/Executing/Executor.cs

```diff
@@ -128,6 +141,19 @@ private async Task<VariableSet> GetVariables(long variableSetId, Dictionary<stri
             return default;
         }
 
+        private async Task<Common.Models.File> GetArchive(string fileName, string version = null, CancellationToken cancellationToken = default)
```

Comment:
It's better to extract all of the interaction with the backend to the separate service

Reply:
done